src/globals/internal-env.php

```diff
@@ -45,5 +45,6 @@
 GlobalEnvManager::putenv('SOURCE_PATH=' . SOURCE_PATH);
 GlobalEnvManager::putenv('DOWNLOAD_PATH=' . DOWNLOAD_PATH);
 GlobalEnvManager::putenv('CPU_COUNT=' . CPU_COUNT);
+GlobalEnvManager::putenv('SPC_ARCH=' . php_uname('m'));
```

Comment:
Why do we have SPC_ARCH, GNU_ARCH and MAC_ARCH? Isn't it just x64 vs aarch64?

Reply:
`SPC_ARCH` is directly from `php_uname('m')`, and on macOS, apple sillicon, is `arm64` while gnu arch on linux is `aarch64`. And Windows will get `AMD64` on x86 platform.

Reply:
Because of these different situation, spc internal always use gnu arch as prefix or suffix, but sometimes need to sync to uname (like `--host=arm64-darwin`)